src/config/constraints.py

```diff
@@ -1,4 +1,8 @@
-CONSTRAINTS = {
+from typing import Any, Dict
+
+# https://stackoverflow.com/questions/54786574/mypy-error-on-dict-of-dict-value-of-type-object-is-not-indexable
```

Comment:
jaki jest sens tego komentarza?

Reply:
miałem problem, bo wyrzucało mi błąd `Value of type "object" is not indexable` i musiałem podac typ - użyłem Any, ale to jest do poprawy bo nie chciałem się bawić w szukanie odpowiendiego typu